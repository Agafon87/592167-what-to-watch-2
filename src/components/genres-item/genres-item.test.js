import React from "react";
import renderer from "react-test-renderer";
import GenresItem from "./genres-item";

it(`GenresItem correctly renderer after test`, () => {
  const tree = renderer.create(<GenresItem films={[]}/>).toJSON();
  expect(tree).toMatchSnapshot();
});
