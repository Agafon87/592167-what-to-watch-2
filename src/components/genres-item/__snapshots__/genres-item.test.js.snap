// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GenresItem correctly renderer after test 1`] = `
<li
  className="catalog__genres-item"
>
  <a
    className="catalog__genres-link"
    href="#"
    onClick={[Function]}
  >
    All genres
  </a>
</li>
`;
